### PR TITLE
LRQA-66428

### PR DESCRIPTION
**Pull request by thaisfrota**
[LRQA-66428](https://issues.liferay.com/browse/LRQA-66428)

### CODE REVIEW

**Comment by thaisfrota** [comment]
ci:test:sf

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase [hunk 1]
```diff
@@ -215,6 +215,124 @@ definition {
...
+
+		WebContent.viewWithStructureCP(
+			skipGotoEditCP = "true",
+			webContentDate = "03/05/2021",
+			webContentLocaleFieldLabel = "ca-ES",
+			webContentNumeric = "20",
+			webContentText = "This is a Text field - ca-ES",
+			webContentTitle = "Web Content Title - ca-ES");
+
+		Navigator.gotoBack();
```
Comment (emanuelfisilva) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase]: Hello @thaisfrota , your test should end here.

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase [hunk 2]
```diff
@@ -215,6 +215,124 @@ definition {
...
+
+		ApplicationsMenu.gotoPortlet(
```
Comment (emanuelfisilva) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase]: Please, create a new macro inside `DataEngine.macro` called `tearDownLocalization` with these instructions to reset default instance language to English US.

**Review by emanuelfisilva — CHANGES_REQUESTED** [review]

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase (hunk 1 above)
Comment (thaisfrota) [reply modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase]: Done.

**Review by emanuelfisilva — APPROVED** [review]

**Comment by emanuelfisilva** [comment]
LGTM
Thanks @thaisfrota

**Comment by emanuelfisilva** [comment]
ci:forward

**Pull request closed without merging**